### ISSUE TITLE
Alignment bug for "completing streak" screen

### ISSUE THREAD

**Issue by jtamiace** [learningequality/ka-lite#3103]
Logged in as a student user

EDIT:
To reproduce: log in as a student and do exercises on an exercise you haven't done before until you see this screen pop up

![congrats](https://cloud.githubusercontent.com/assets/6668144/6339932/34349dac-bb6f-11e4-9142-f625cd05951f.PNG)

**Comment by MCGallaspy** [comment]
Can you give a more detailed explanation of how to reproduce this?

**Comment by jesumer** [comment]
Hi @MCGallaspy,

I review this issue and tried 3 exercises but I don't see anything strange here. cc@jtamiace 
![screen shot 2015-02-24 at 9 50 22 am](https://cloud.githubusercontent.com/assets/2810009/6341877/da05e5c4-bc0a-11e4-85cf-f3072140fb63.png)

Please give more detailed explanation to reproduce this as what @MCGallaspy told.

**Comment by cpauya** [comment]
@jesumer It's a different message, did you do an exercise that you haven't done before as @jtamiace mentioned above?

**Comment by jesumer** [comment]
@cpauya. Yes. I did as what is my screenshot. That is what the output when I tried the exercises. In short I can't reproduce the issue since it is not clear how to reproduce. I need the clear explanation on how to reproduce.

**Comment by MCGallaspy** [comment]
@jtamiace can you help us reproduce this?

**Comment by cpauya** [comment]
I can't reproduce this too.  But getting a hint from the screenshot, I completed one exercise then took another one to be on a streak, I got this for the second exercise:

![screenshot 2015-02-24 10 27 08](https://cloud.githubusercontent.com/assets/175580/6342187/fc8533fc-bc0f-11e4-8f93-439ee4c69545.png

Then after answering the 8th question, I got this
![screenshot 2015-02-24 10 30 29](https://cloud.githubusercontent.com/assets/175580/6342219/55e257f4-bc10-11e4-8d20-95fca0a213d8.png)

Is this expected behaviour @MCGallaspy?  If so then, this is not an issue I think.
